Author: Sune Vuorela
Description: Ensures we link with rt on all the platforms where we 
have glibc. Not yet upstreamed. Requires a upstreamming of the gnukfreebsd patch
--- a/Source/WebKit2/WebKit2.pri
+++ b/Source/WebKit2/WebKit2.pri
@@ -71,7 +71,7 @@ INCLUDEPATH += $${ROOT_WEBKIT_DIR}/Sourc
 
 INCLUDEPATH += $$WEBKIT2_GENERATED_SOURCES_DIR
 
-linux-*: {
+linux-*:!android|hurd-*|gnukfreebsd-* {
     # -lrt is required for shm_open and shm_unlink.
     LIBS += -lrt
 }
